core/src/elua_manager.cpp

```diff
@@ -0,0 +1,51 @@
+#include "qabstractbutton.h"
+#include <QCoreApplication>
+#include <QMessageBox>
+#include <elua_manager.h>
+
+#include <pluginbase/preferences.h>
+
+#include <QFile>
+#include <QPushButton>
+#include <QTextBrowser>
+
+using namespace scopy;
+
+void ELUAManager::openELUAMessageBox(QMainWindow* window) {
+	QMessageBox* msgBox = new QMessageBox(window);
+
+	QSize mSize = msgBox->sizeHint();
+	QRect screenRect = window->geometry();
+
+	msgBox->setTextFormat(Qt::RichText);
+	msgBox->setText(getELUA());
+	msgBox->setTextInteractionFlags(Qt::LinksAccessibleByMouse);
+
+	QPushButton* yesButton = new QPushButton("Yes");
+	msgBox->addButton(yesButton ,QMessageBox::AcceptRole);
+
+	QPushButton* noButton = new QPushButton("No");
+	msgBox->addButton(noButton ,QMessageBox::RejectRole);
+
+	msgBox->setModal(true);
+	msgBox->show();
```

Comment:
Make sure that the license message box appears centered on top of the opened application.